llvm/include/llvm/SYCLLowerIR/LowerESIMD.h

```diff
@@ -10,8 +10,8 @@
 // - ESIMD intrinsics, e.g.:
 //     template <typename Ty, int N, int M, int VStride, int Width,
 //       int Stride, int ParentWidth = 0>
-//       sycl::intel::gpu::vector_type_t<Ty, M>
-//       __esimd_rdregion(sycl::intel::gpu::vector_type_t<Ty, N> Input,
+//       sycl::ext::intel::gpu::vector_type_t<Ty, M>
```

Comment:
Since you are fixing this anyway, could you please change 'sycl::ext::intel::gpu' to 'sycl::ext::intel::experimental::esimd' here and below?

Reply:
Changed in https://github.com/intel/llvm/pull/4014/commits/5f8dfa54758be4bc71d9169091a22e45d449a961